packages/changed-elements-react/src/common/types.ts

```diff
@@ -0,0 +1,12 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { BeEvent } from "@itwin/core-bentley";
+
+export type ReactComponentLifeCycle = "mounted" | "updating" |"unmounted";
+export type EventActionTuple = {
```

Comment:
If this is only used in the widget, does it make sense to be exporting it, and adding it to a common folder? I think this shouldn't be exposed from the package if its only used internally once.